package.json

```
@@ -1,9 +1,9 @@
 {
-  "name": "sibmei4",
-  "version": "4.1.0",
+  "name": "sibmei",
+  "version": "5.0.0",
```

Maybe keep the plugin in sync with the MEI version? 

```suggestion
  "version": "5.1.0",
```

---

I thought about that, but I don't think that's a good idea because we would only be able to make a release with a sensible version number when there's a new MEI version. For example, we're planning another Sibmei 4 release although there's no new MEI version. I think it's a good idea to keep the major version number in sync with the MEI schema and use our own schema for the rest.

---

Does the update to MEI 5 even justify the jump to a new major version number? I think the last release had more substancial changes than this. If we want to move to version 5 the main point here is probably dropping libmei …

---

#142 

We discussed this earlier. Verovio also keeps its major version numbers in sync with MEI.